make/common/modules/LauncherCommon.gmk

```diff
@@ -45,7 +45,6 @@ endif
 
 LAUNCHER_SRC := $(TOPDIR)/src/java.base/share/native/launcher
 LAUNCHER_CFLAGS += -I$(TOPDIR)/src/java.base/share/native/launcher \
-    -I$(TOPDIR)/src/java.desktop/share/native/include \
```

Comment:
This line was added recently, not sure why.

Reply:
Hmm .. IIRC Alex Zuev said  there is a (closed) native test and adding this was the only way he could make that test work .. 
Now clearly this location no longer makes sense when the file is moved, but someone will need to do something about the test. It needs Alex to chime in here.

Reply:
Superficially, it looks like the closed test has a direct pointer to where to get this which would need to be re-adjusted
but now I really do need Alex to explain why the above include dir was needed ..

Reply:
> Superficially, it looks like the closed test has a direct pointer to where to get this which would need to be re-adjusted

The test has direct link to the include yet i had to add this line so jtreg could find it in the build phase. The fact that it will lead to publishing of this include was not an intentional consequence and it needs to be changed, probably the test should be adjusted later if it will start failing on certain platforms.